Resources/gulpfile.js

```diff
@@ -0,0 +1,21 @@
+const gulp = require('gulp');
+const babel = require('gulp-babel');
+const rename = require('gulp-rename');
+const uglify = require('gulp-uglify');
+const wrap = require('gulp-wrap');
+
+gulp.task('js', () => {
+    gulp.src('js/router.js')
```

Comment:
should have a `return` statement, so that Gulp is notified about the result too (to know when it end, and whether it succeeded)

Reply:
Done in a soon-to-be pushed commit.